.travis.yml

```diff
@@ -90,7 +88,15 @@ script:
   - if [[ $DEVITO_BACKEND != 'yask' ]]; then py.test -vs --nbval examples/seismic/tutorials; fi
   - if [[ $DEVITO_BACKEND != 'yask' ]]; then py.test -vs --nbval examples/cfd; fi
 
-  # Docs generation and deployment
+  # Docs generation
   - sphinx-apidoc -f -o docs/ examples
   - sphinx-apidoc -f -o docs/ devito devito/yask/*
-  - if [[ $DEVITO_ARCH == 'gcc-5' ]]; then ./docs/deploy.sh; fi
+  # Docs deployment only once from a single build
+  - if [[ $DEVITO_ARCH == 'gcc-5' ]]; then
+      set -e;
+      pip install doctr;
+      cd docs;
```

Comment:
Should we move the `cd docs; make html; cd ..` out of the conditional clause and run it on all builders? This might enable us to capture YASK-related sphinx issues and add the backedn docs back again.

Reply:
yes

Reply:
done